Automattic/wp-calypso
# Design Picker: Retire image verticalization

Related to #75666

## Proposed Changes

This PR removes code related to the image verticalization in the onboarding design picker. Since we are only targeting the onboarding design picker first, general components/flows such as design preview or site setup have been left intact, in case other flows might want to, or are already using it.

## Testing Instructions



* Head to the Goals screen and continue until you've landed in the Vertical screen.
* Select one of the Verticals in the dropdown, such as "Food".
* Continue to the Onboarding Design Picker.
* Check that the following themes have the default images, and not the verticalized ones.
  * Arbutus.
  * Geologist.
  * Stewart.
  * Zoologist. 
* Select one of those themes and ensure that the theme activation works as before.
* (Optional) Try the same steps without selecting any vertical.
* (Optional) Try the same steps selecting another theme that's not one of the four mentioned above. 

## Pre-merge Checklist
